Description: use 0711, not 0755, as the default perms for storage dirs
 That is what we want for /var/lib/libvirt/images.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: no
diff -Naurp libvirt-1.2.0.orig/src/conf/storage_conf.c libvirt-1.2.0/src/conf/storage_conf.c
--- libvirt-1.2.0.orig/src/conf/storage_conf.c	2013-11-25 20:56:37.000000000 -0500
+++ libvirt-1.2.0/src/conf/storage_conf.c	2013-12-02 12:08:16.316555557 -0500
@@ -48,7 +48,7 @@
 
 #define VIR_FROM_THIS VIR_FROM_STORAGE
 
-#define DEFAULT_POOL_PERM_MODE 0755
+#define DEFAULT_POOL_PERM_MODE 0711
 #define DEFAULT_VOL_PERM_MODE  0600
 
 VIR_ENUM_IMPL(virStorageVol,
diff -Naurp libvirt-1.2.0.orig/tests/storagepoolxml2xmltest.c libvirt-1.2.0/tests/storagepoolxml2xmltest.c
--- libvirt-1.2.0.orig/tests/storagepoolxml2xmltest.c	2013-11-25 20:56:37.000000000 -0500
+++ libvirt-1.2.0/tests/storagepoolxml2xmltest.c	2013-12-02 13:04:35.904636133 -0500
@@ -94,7 +94,6 @@ mymain(void)
     DO_TEST("pool-iscsi");
     DO_TEST("pool-iscsi-auth");
     DO_TEST("pool-netfs");
-    DO_TEST("pool-netfs-gluster");
     DO_TEST("pool-scsi");
     DO_TEST("pool-scsi-type-scsi-host");
     DO_TEST("pool-scsi-type-fc-host");
